docker/docker-compose-base.yml

```diff
@@ -1,13 +1,15 @@
 services:
     app:
-      image: ghcr.io/openboxes/openboxes:feature-upgrade-to-grails-3.3.10
+      image: ${OB_IMAGE_REPOSITORY-ghcr.io/}openboxes/openboxes:${OB_VERSION:-latest}
       container_name: openboxes-app
       expose:
         - "8080"
+      ports:
+        - 8080:8080
       environment:
         DATASOURCE_USERNAME: ${DATASOURCE_USERNAME:-openboxes}
         DATASOURCE_PASSWORD: ${DATASOURCE_PASSWORD:-openboxes}
-        DATASOURCE_URL: ${DATASOURCE_URL:-jdbc:mysql://db/openboxes?useSSL=false}
+        DATASOURCE_URL: ${DATASOURCE_URL:-jdbc:mysql://db/openboxes?serverTimezone=UTC&useSSL=false}
```

Comment:
another "bug" fix. Without setting `serverTimezone=UTC` we get timezone weirdness. This puts us in sync with what we do for the non-docker datasource url